nesta/packages/nih/tests/test_nih_abstract_yielder.py

```diff
@@ -19,7 +19,7 @@ def setUp(self):
         session = Session()
         Base.metadata.drop_all(self.engine)
         Base.metadata.create_all(self.engine)
-        for i in range(0, self.n_abstracts):
+        for i in range(1, self.n_abstracts+1):
```

Comment:
This is a consequence of doing `autoincrement=False`, since `0` is interpretted as meaning "increment me" by MySQL